metecho/api/models.py

```diff
@@ -530,6 +536,15 @@ def finalize_status_completed(self, pr_number, *, originating_user_id):
         self.save()
         self.notify_changed(originating_user_id=originating_user_id)
 
+    def queue_available_task_org_config_names(self, user):
+        from .jobs import available_task_org_config_names_job
+
+        available_task_org_config_names_job.delay(self, user=user)
+
+    def finalize_available_task_org_config_names(self, originating_user_id=None):
+        self.save()
+        self.notify_changed(originating_user_id=originating_user_id)
```

Comment:
@wlonk If this is going to happen in a job (does it actually need to happen in a job?), I think we'll need to use our `currently_*` pattern so that the front end knows when it's done.

Reply:
It needs to happen in a job because it needs to check out code from GitHub. I'll add the `currently_*` pattern.